k8s/perftest/common-overlay/wa/kustomization.yaml

```diff
@@ -5,8 +5,11 @@ bases:
 - ../../../namespaces/wa
 - ../../../namespaces/wa/namespace.yaml
 - ../../../namespaces/all-namespaces/nonprod-role.yaml
+- ../../../namespaces/wa/wa-task-batch-termination/wa-task-batch-termination.yaml
+- ../../../namespaces/wa/wa-task-batch-initiation/wa-task-batch-initiation.yaml
```

Comment:
Did you need to do a common-overlay for perftest?  I thought we would want to deploy the `master` code there.
It's no problem, it's just to understand.